player_state.cpp

```diff
@@ -39,5 +41,20 @@ void test_player_state()
     assert(to_str(player_state::stunned) == "stunned");
   }
   #endif // FIX_ISSUE_276
+
+  #define FIX_ISSUE_509
```

Comment:
After the test passes, you can remove the `#define`, `#ifndef` and `#ifdef` lines. It is optional as I use that for newbie Issues :-)